.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
+
+[*]
+insert_final_newline = true
```

Comment:
is eol also a good idea here?

Reply:
the doc says to not set eol if we want native line endings